server/src/services/UserServices.js

```diff
@@ -4,7 +4,7 @@ const EmailValidator = require('email-validator');
 const UserModel = require('../models/UserModel');
 const newUserSubscription = require('../subscribers/newUserSubscription');
 
-const passwordValidator = /(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[^A-Za-z0-9])(?=.{8,})/;
+const passwordValidator = /(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.{8,})/;
```

Comment:
Please revert this change to what it was previously - we do want to be checking for a password that has at least one special character, and this regex does not work for that. For example, `testADA123` matches this regex, but should not be a valid password in our system